examples/intel/fileFromFilepath.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+import pangea.exceptions as pe
+from pangea.config import PangeaConfig
+from pangea.services import FileIntel
+
+token = os.getenv("PANGEA_FILE_INTEL_TOKEN")
```

Comment:
One thing I'd like for the SDK team to start standardizing over the SDKs is the ways we do auth. I think boto3 does a pretty good job of it where they have a priority list of

1. environment variables called AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
2. file stored in ~/.aws/credentials
3. introspected from stuff like ec2 instance profile
4. introspected from stuff like an OIDC provider

Obviously 3 and 4 don't really make sense for us, but I think we can always start doing 1 and 2